src/tests/ftest/dfuse/container_type.py

```diff
@@ -31,40 +33,46 @@ def test_dfuse_container_check(self):
         :avocado: tags=DfuseContainerCheck,test_dfuse_container_check
         """
         # get test params for cont and pool count
-        cont_types = self.params.get("cont_types", '/run/container/*')
+        cont_types = self.params.get('cont_types', '/run/container/*')
 
         # Create a pool and start dfuse.
-        self.add_pool(connect=False)
+        self.log_step('Creating a single pool')
+        pool = self.get_pool(connect=False)
 
         for cont_type in cont_types:
+            description = f"{cont_type if cont_type == 'POSIX' else 'default'}"
```

Comment:
dfuse only works with POSIX, so I recommend
```suggestion
            description = f"{cont_type if cont_type == 'POSIX' else 'non-POSIX'}"
```

Reply:
Done